apps/condo/domains/acquiring/constants/errors.js

```diff
@@ -0,0 +1,30 @@
+const REGISTER_MP_EMPTY_INPUT = '[groupedReceipts:empty] GroupedReceipts was empty'
+const REGISTER_MP_EMPTY_RECEIPTS = '[groupedReceipts:receiptsIds:empty] Each group of receipts should contains at least 1 receipt'
+const REGISTER_MP_CONSUMERS_DUPLICATE = '[groupedReceipts:consumerId:duplicate] There are some groupedReceipts with same consumerId'
+const REGISTER_MP_RECEIPTS_DUPLICATE = '[groupedReceipts:consumerId:duplicate] Found duplicated receiptsIds. Note, that single receipts can only occur in one ServiceConsumer per and cannot be noticed twice'
+const REGISTER_MP_REAL_CONSUMER_MISMATCH = '[groupedReceipts:consumerId:nonExistingConsumer] Cannot find all specified ServiceConsumers.'
+const REGISTER_MP_NO_ACQUIRING_CONSUMERS = '[serviceConsumer:noAcquiringContext] Some of ServiceConsumers doesn\'t have Acquiring context.'
+const REGISTER_MP_MULTIPLE_INTEGRATIONS = '[serviceConsumer:acquiringContext:multipleIntegrations] Listed consumerIds are linked to different acquiring integrations'
+const REGISTER_MP_CANNOT_GROUP_RECEIPTS = '[acquiringIntegration:canGroupReceipts:false:multipleReceipts] receiptsIds total length was > 1, but acquiring integration cannot group billing receipts'
+const REGISTER_MP_UNSUPPORTED_BILLING = '[acquiringIntegration:supportedBillingIntegrations:notIncludesReceiptBilling] Some of billing receipts is not supported by ServiceConsumer\'s acquiring integration'
+const REGISTER_MP_REAL_RECEIPTS_MISMATCH = '[groupedReceipts:receiptsIds:nonExistingReceipt] Cannot find all specified BillingReceipts.'
+const REGISTER_MP_DELETED_RECEIPTS = '[groupedReceipts:receiptsIds:deletedReceipt] Cannot pay for deleted receipts.'
+const REGISTER_MP_MULTIPLE_CURRENCIES = '[groupedReceipts:receiptsIds:context:integration:currencyCode:multipleFound] Cannot pay for receipts with multiple currencies.'
+const REGISTER_MP_BILLING_ACCOUNTS_NO_MATCH = '[groupedReceipts:receiptsIds:noCommonBillingAccount] Billing receipt with specified id doesn\'t have common billing account with specified ServiceConsumer'
+
+
+module.exports = {
+    REGISTER_MP_EMPTY_INPUT,
+    REGISTER_MP_EMPTY_RECEIPTS,
+    REGISTER_MP_CONSUMERS_DUPLICATE,
+    REGISTER_MP_RECEIPTS_DUPLICATE,
+    REGISTER_MP_REAL_CONSUMER_MISMATCH,
+    REGISTER_MP_NO_ACQUIRING_CONSUMERS,
+    REGISTER_MP_MULTIPLE_INTEGRATIONS,
+    REGISTER_MP_CANNOT_GROUP_RECEIPTS,
+    REGISTER_MP_UNSUPPORTED_BILLING,
+    REGISTER_MP_REAL_RECEIPTS_MISMATCH,
+    REGISTER_MP_DELETED_RECEIPTS,
+    REGISTER_MP_MULTIPLE_CURRENCIES,
+    REGISTER_MP_BILLING_ACCOUNTS_NO_MATCH,
+}
```

Comment:
It's generally better to explicitly point user to entity with error: 

Not `Cannot pay for deleted receipts.` but `Receipt with id {id} is deleted - cannot pay for deleted receipts.`

There are a lot of errors in this custom mutation and this change would ease the pain of debugging and making it work

Reply:
But this is not critical :)

Reply:
Let me explain. This is just common prefixes. Like "type" of error. Additional information are provided then errors are thrown (see this https://github.com/open-condo-software/condo/pull/780#discussion_r745374715 as example)

Reply:
Errors without specifying any of ids needed to catch them in tests (see `expectToThrowMutationError` from `@condo/domains/common/utils/testSchema/index`)

Reply:
Ok